repository-data/webfiles/src/main/resources/site/freemarker/hee/pages/minihub-main.ftl

```diff
@@ -81,6 +81,9 @@
                         </#if>
                     </ul>
                 </nav>
+                <div class="nhsuk-width-container">
```

Comment:
Again, could we wrap this up with `<#if document.relatedContent??>...</#if>` as `guidanceDocument` is an optional field ?

Reply:
done.